UnityProject/Assets/Scripts/Tilemaps/TileManager.cs

```diff
@@ -82,8 +82,6 @@ private void Awake()
 #if UNITY_EDITOR
 		CacheAllAssets();
 #endif
-
-		if (!initialized) StartCoroutine(LoadAllTiles());
```

Comment:
Could you provide a bit of background on what this did / why removed? Seems important

Reply:
looks like Merge conflict again